core/src/test/java/org/infinispan/executors/BlockingTaskAwareExecutorServiceTest.java

```diff
@@ -53,33 +47,24 @@ public boolean isSatisfied() throws Exception {
    public void testMultipleExecutions() throws Exception {
       BlockingTaskAwareExecutorServiceImpl executorService = createExecutorService();
       try {
-         List<DoSomething> tasks = new LinkedList<DoSomething>();
+         List<DoSomething> tasks = new LinkedList<>();
 
          for (int i = 0; i < 30; ++i) {
             tasks.add(new DoSomething());
          }
 
-         for (DoSomething doSomething : tasks) {
-            executorService.execute(doSomething);
-         }
+         tasks.forEach(executorService::execute);
 
          for (DoSomething doSomething : tasks) {
             assert !doSomething.isReady();
             assert !doSomething.isExecuted();
          }
 
-         for (DoSomething doSomething : tasks) {
-            doSomething.markReady();
-         }
+         tasks.forEach(BlockingTaskAwareExecutorServiceTest.DoSomething::markReady);
```

Comment:
the IDEA puts a yellow line with a warning with it :) and it is more annoying lol